InnerEye/ML/SSL/lightning_containers/ssl_container.py

```diff
@@ -223,25 +225,39 @@ def _get_transforms(self, augmentation_config: Optional[CfgNode],
         examples.
         :param dataset_name: name of the dataset, value has to be in SSLDatasetName, determines which transformation
         pipeline to return.
-        :param is_ssl_encoder_module: if True the transformation pipeline will yield two version of the image it is
-        applied on. If False, return only one transformation.
+        :param is_ssl_encoder_module: if True the transformation pipeline will yield two versions of the image it is
```

Comment:
I don't get the description of this argument (the original version was similarly cryptic)

Reply:
Also, the name of the argument is somewhat odd. It seems to refer to how the argument is used, rather than what it does. At other places, this is return_two_views_per_sample

Reply:
I completely agree on the parameter name...it took me a while to understand what this was doing. I was hoping the description could help. The problem is that this boolean is somehow used for different things and in several places. Changing the name only in this method would create confusion, we would need to change it in all the module I think

The key is that if the data module is a SSL encoder we want to return both two views per sample and use augmentations in the validation, if instead the data module is a linear head we don't want neither of the two. If you have suggestions on how to better name this parameter I'd be happy to change it